stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -69,6 +70,13 @@ public class MessageListViewModel(
      */
     private var threadListData: MessageListItemLiveData? = null
 
+    /**
+     * Regulates the visibility of deleted messages.
+     */
+    @InternalStreamChatApi
+    public var deletedMessagesVisibility: MutableLiveData<DeletedMessagesVisibility> =
```

Comment:
🤔  Do we need to keep it public? We are using it inside `bindView` method, but maybe we can replace direct`deletedMessagesVisibility .value` usage with`setDeletedMessagesVisibility ` method call?

Reply:
Agreed